platform-coop-toolkit/coop-library-framework
# Add short title (resolves #105)

## Description

Adds an optional field for a short title.

## Steps to test

1. Add a resource and enter a short title.
2. Save.

**Expected behavior:** The short title field saves to metadata.

## Additional information

Not applicable.

## Related issues

- Resolves #105
- Depends on #110
